CustomApps/lyrics-plus/manifest.json

```diff
@@ -76,6 +76,9 @@
         "OptionsMenu.js",
         "TabBar.js",
         "Utils.js",
-        "Settings.js"
+        "Settings.js",
+        "Translator.js",
+        "dict_part_1.js",
+        "dict_part_2.js"
```

Comment:
well... i would rather load these dict files when it's needed. 30mb is a lot, since it adds another megabytes to download for people, even when they don't want it

Reply:
Yes that's quite understandable, how about making it so that there's a download button when first opening the conversion menu on a Japanese song? The dicts will be cached locally after that.

Reply:
Since both the dictionary files are already inside the custom app folder, this means that it will be included in CLI releases and users would only need to download them whenever they upgrade the CLI. Sounds reasonable to me tbh, but it still means that the CLI package is now 10 times larger than it ever was:
![image](https://user-images.githubusercontent.com/77577746/197366688-b5a4eeb6-f15f-4b2b-9437-e716331b1cc0.png)

If we don't want to do this, then the only option left is to *not* include it into CLI releases and use it as a remote script via `jsdelivr` like before. This would mean that users can prompt when to download the dictionary, but the client would need to do it every time cache is dumped.

Reply:
To be entirely honest I think this is the best solution, multiplying the cli size by 10 does not seem worth it to me. Particularly since a majority of people probably won't ever do so much as touch this feature. Or, perhaps we could merge the cli changes but make the modified lyrics plus it's own separate repo. (Kind of similar to when we made it so custom app owners could run code like an extension for marketplace, instead of building marketplace into the cli repo like expFeatures or home/sidebar config.)

Reply:
Ah, just checked back and saw that there are actually no cli changes, why not just make this a separate repo altogether?